rryb77/deck-tavern-capstone
# Player Class - Added

## Changes:

- Added a UserProvider.js to add/retrieve user accounts that contain a username and email
- Added an auth system via Login.js / Register.js for user registering/login
- Added a deck builder view in ApplicationViews
- Added DeckTavern.js to handle the base layout depending on whether a user is logged in or not
- Added a NavBar.js to handle the navigation of the site
- Added PlayerClassList.js to handle application state to get a base rendered to the DOM
- Added PlayerClassCard.js to render hero class images are rendered to the DOM
  - Hero class images are stored in the public images folder
- Added PlayerClassProvider.js to obtain hero classes from the JSON server.

## To Test:

1. Serve up the app
2. Serve up the JSON
3. Register/Login
4. Click the "Deck Builder" link in the navbar
5. Make sure hero images are showing up in the DOM
6. Logout
